composer.json

```diff
@@ -1,8 +1,8 @@
 {
 	"require": {
 		"mediawiki/oauthclient": "1.2.0",
-		"symfony/process": "^5.1",
-		"symfony/yaml": "^5.1",
+		"symfony/process": "^5.4",
+		"symfony/yaml": "^5.4",
```

Comment:
The $env feature has been supported for ages in Process, but for some reason it wasn't working in 5.1 and is now in 5.4 (6.0 is out but requires PHP8).